modules/core/src/test/scala/ExampleTests.scala

```diff
@@ -32,6 +32,30 @@ class ExampleTests extends munit.FunSuite, TerminalTests:
     Next.Stop
   )
 
+  terminalTestComplete("alternatives.infiniscroll.single")(
+    Prompt.SingleChoice(
+      "How was your day?",
+      List(
+        "amazing",
+        "productive",
+        "relaxing",
+        "stressful",
+        "exhausting"
```

Comment:
Can you please add a test which also has some typing? To make sure that filtering doesn't screw up the pagination